packages/components/package.json

```diff
@@ -1,89 +1,91 @@
-        "sass-resources-loader": "^2.1.1",
-        "style-loader": "^1.2.1",
-        "svg-sprite-loader": "^5.2.1",
-        "svgo-loader": "^3.0.0",
-        "webpack": "^5.46.0",
-        "webpack-bundle-analyzer": "^4.3.0",
-        "webpack-cli": "^4.7.2"
-    },
-    "dependencies": {
-        "@contentpass/zxcvbn": "^4.4.3",
-        "@deriv/deriv-onboarding": "^1.0.0",
-        "@deriv/shared": "^1.0.0",
-        "@storybook/addon-knobs": "^6.1.0",
-        "@wapps/storybook-addon-state": "0.0.4",
-        "classnames": "^2.2.6",
-        "gh-pages": "^2.1.1",
-        "lodash.throttle": "^4.1.1",
-        "prop-types": "^15.7.2",
-        "react-content-loader": "^4.3.2",
-        "react-div-100vh": "^0.3.8",
-        "react-dom": "^16.14.0",
-        "react-drag-drawer": "^3.3.4",
-        "react-dropzone": "11.0.1",
-        "react-pose": "^4.0.10",
-        "react-router-dom": "^5.2.0",
-        "react-swipeable": "^5.5.1",
-        "react-tiny-popover": "^5.1.0",
-        "react-transition-group": "^4.3.0",
-        "react-virtualized": "^9.22.2",
-        "@storybook/addons": "^6.1.0",
-        "@storybook/react": "^6.1.0",
-        "@storybook/addon-info": "^5.2.4",
-        "@storybook/addon-actions": "^6.1.0",
-        "@storybook/addon-links": "^6.1.0",
-        "glob": "^7.1.5"
-    }
+  "name": "@deriv/components",
+  "version": "1.0.0",
+  "description": "deriv components",
+  "main": "src/index.js",
+  "private": true,
+  "engines": {
+    "node": "^14.17.1"
+  },
+  "scripts": {
+    "start": "rimraf lib && npm run test && npm run serve",
+    "serve": "rimraf lib && cross-env BUILD_MODE='serve' webpack --mode=production --progress --watch",
+    "build": "rimraf lib && npm run build:icons && webpack --mode=production --progress && npm run build:icons-manifest",
+    "build:icons": "node -e 'require(\"./utils/build-icons\").buildIcons()'",
+    "build:icons-manifest": "node -e 'require(\"./utils/build-icons-manifest\").buildIconsManifest()'",
+    "build:travis": "rimraf lib && webpack --mode=production",
+    "test": "node \"./src/components/icon/__tests__/icons.js\"",
+    "test:eslint": "eslint \"./src/**/*.?(js|jsx)\"",
+    "deploy": "echo \"No deploy specified\"",
+    "deploy:clean": "echo \"No deploy:clean specified\"",
+    "deploy:folder": "echo \"No deploy:folder specified\"",
+    "deploy:staging": "echo \"No deploy:staging specified\"",
+    "deploy:production": "echo \"No deploy:production specified\"",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "author": "Deriv",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@babel/core": "^7.12.10",
+    "@babel/eslint-parser": "^7.17.0",
+    "@babel/preset-react": "^7.16.7",
+    "@storybook/addon-actions": "^6.5.10",
+    "@storybook/addon-essentials": "^6.5.10",
+    "@storybook/addon-interactions": "^6.5.10",
+    "@storybook/addon-links": "^6.5.10",
+    "@storybook/builder-webpack4": "^6.5.10",
```

Comment:
do we need `@storybook/builder-webpack4` and `@storybook/manager-webpack4`? 
since we are using the webpack5 versions now.

Reply:
no we don't need them